### PR TITLE
(Potentially breaking, do not merge) Fix timers

### PR DESCRIPTION
**Pull request by hughesjeff**
PLEASE bug test this and let me know if it works. 

DO NOT MERGE until we have verified this works.

### CODE REVIEW

File: app/src/main/java/xyz/jhughes/laundry/notificationhelpers/NotificationCreator.java [hunk 1]
```diff
@@ -55,7 +59,7 @@ public void onFinish() {
     private static void updateTimeNotification(String machine, Context context, long timeLeft) {
         int id = notifcationIds.get(machine);
         String countDown;
-        System.out.println(timeLeft);
+        //System.out.println(timeLeft);
```
Comment (vidia) [inline app/src/main/java/xyz/jhughes/laundry/notificationhelpers/NotificationCreator.java]: Delete pls.

File: app/src/main/java/xyz/jhughes/laundry/notificationhelpers/NotificationCreator.java [hunk 2]
```diff
@@ -25,9 +25,12 @@
     private static HashMap<String, Integer> notifcationIds = new HashMap<>();
     private static HashMap<Integer, CountDownTimer> timers = new HashMap<>();
     private static int id = 0;
+    private static NotificationCreator self;
 
     @Override
     public int onStartCommand(Intent intent, int flags, int startId) {
+        self = this;
```
Comment (vidia) [inline app/src/main/java/xyz/jhughes/laundry/notificationhelpers/NotificationCreator.java]: Why does this fix it? I don't know services well enough to know.

File: app/build.gradle [hunk 3]
```diff
@@ -8,14 +8,14 @@ repositories {
...
-    buildToolsVersion '24.0.2'
+    buildToolsVersion '25.0.0'
 
     defaultConfig {
         applicationId "xyz.jhughes.laundry"
         minSdkVersion 16
         targetSdkVersion 24
-        versionCode 9
-        versionName "2.3"
+        versionCode 10
```
Comment (vidia) [inline app/build.gradle]: Wait. Why is this change on your branch? Are you preemptively upticking it for a release?

**Review by vidia — DISMISSED** [review]
Looks good but I don't understand how this fixes it. I'm curious.

File: app/src/main/java/xyz/jhughes/laundry/notificationhelpers/NotificationCreator.java (hunk 2 above)
Comment (hughesjeff) [reply app/src/main/java/xyz/jhughes/laundry/notificationhelpers/NotificationCreator.java]: That's just so I can get `this` in the static function for removing timers.

File: app/build.gradle (hunk 3 above)
Comment (hughesjeff) [reply app/build.gradle]: Android Studio must've just done that.

**Comment by vidia** [comment]
Addressing #79, adding for reference/docs.

**Comment by hughesjeff** [comment]
Fixes #79

**Comment by hughesjeff** [comment]
I'm merging this. We had discussed it was working and done before.

**Pull request merged**